src/site/stages/build/process-cms-exports/transformers/page.js

```diff
@@ -0,0 +1,59 @@
+const { flatten, isEmpty } = require('lodash');
+const { getDrupalValue } = require('./helpers');
+
+function createMetaTag(type, key, value) {
+  return {
+    type,
+    key,
+    value,
+  };
+}
+
+function pageTransform(entity) {
+  const transformed = entity;
+  const {
+    title,
+    changed,
+    fieldIntroText,
+    fieldPageLastBuilt,
+    fieldAlert,
+    fieldDescription,
+    moderationState: [{ value: published }],
+    metatag: { value: metaTags },
+  } = entity;
+  // collapse title
+  // Question: Can we always assume that title is an array of one item, with that item being an object with a `value` key?
```

Comment:
We can once we get the [JSON schema validation](https://github.com/department-of-veterans-affairs/vets-website/pull/10982) in place. :wink: